go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/ceph/go-ceph
 
-go 1.19
+go 1.21.6
```

Comment:
Is this change absolutely positively necessary? If not, please leave it as-is.

Reply:
Reverted the version to original (v1.19).

Reply:
Hope these changes are good for the approval. Thanks.